client/gutenberg/editor/components/post-publish-button-or-toggle/index.jsx

```diff
@@ -0,0 +1,225 @@
+/** @format */
+/**
+ * External Dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { get } from 'lodash';
+import { translate } from 'i18n-calypso';
```

Comment:
Mentioned already in https://github.com/Automattic/wp-calypso/pull/29594/files#r242893520, but for clarity:

The [recommended way](https://github.com/Automattic/i18n-calypso/blob/9556007bd2dd6d3bb88edfd0c5e8e4275edf06b4/README.md) of using `i18n-calypso` is through the `localize` HOC.

E.g.
```jsx
import { localize } from 'i18n-calypso';

const FooBar = ( { translate } ) => ( <div>{ translate( 'Howdy!' ) }</div> );

export localize( FooBar );
```